uqcsbot/scripts/advent.py

```diff
@@ -59,10 +58,11 @@ def sort_none_last(key):
 # TODO: make these types more specific with TypedDict and Literal when possible.
 
 class Member:
-    def __init__(self, name: str, score: int, stars: int) -> None:
+    def __init__(self, name: str, score: int, stars: int, globo: int) -> None:
         self.name = name
         self.score = score
         self.stars = stars
+        self.globo = globo
```

Comment:
Nit: It would be nice to replace "self.score" with "self.local" and "self.globo" with "self.global" (as mentioned above).